internal/util/types.go

```diff
@@ -0,0 +1,8 @@
+package util
+
+type contextKey string
+
+// Context keys.
+const (
+	ContextKeyLogger = contextKey("logger")
```

Comment:
this pkg is nor useful right now.

skip for now. create new PR where you properly design a `util.WithLogger(ctx, logger)` method.

ContextKeyLogger is an implementation detail that should not be exposed to the outside world.